.github/workflows/workspace.yml

```diff
@@ -20,7 +20,11 @@ jobs:
       - uses: dtolnay/rust-toolchain@master
         with:
           toolchain: stable
-      - run: cargo build --all --all-features
+# Use --tests to include the test code in the compilation, avoiding `unused warning` errors
+# e.g.
+# warning: method `calc_balance` is never used
+#   --> p32/src/lib.rs:138:19
```

Comment:
No, I intentionally asked you yo add this build job so you would fix the unused code warnings. You need to make the appropriate items public in your crate.

Reply:
changed in 87b008592b82e80748a688fd77d57bb54bc48ec4